app/controllers/halls_controller.rb

```diff
@@ -0,0 +1,52 @@
+class HallsController < ApplicationController
+
+    before_action :set_hall, only: [:show, :update, :destroy]
+
+    def index
+        @halls = Hall.all.map do |hall|
+            {
+                name: hall.hall_number,
+                capacity: hall.capacity
+            }
+        end
+        
+        render json: @halls
+    end
+
+    def show
+        @cinema_hall = {
```

Comment:
You use inconsistent formatting for [Hash](https://ruby-doc.org/core-3.0.1/Hash.html). It's decreasing the readability of the file. I highly recommend you to use a linter like [rubocop](https://github.com/rubocop/rubocop) to avoid such inconsistency. 

I also encourage you to take a look at this [styleguide](https://github.com/rubocop/ruby-style-guide#the-ruby-style-guide).